### DISCUSSION TITLE
wglmakie support for phones and tablets

### DISCUSSION THREAD

**Discussion by lazarusA** [JuliaPlots/Makie.jl#4219] [Ideas]
This is just to raise awareness of the fact that even tough one can successfully deploy apps on the browser they seem only to work [zooming and rotating] on the computer and not phones or tablets where one usually uses the touch screen to interact.

- This app https://this-is-fine.org by SimonDanisch if good example for it. 
- a more complicated one could be this one: https://lazarusa.github.io/ClimateVisualisations/population_app.html 😄 

if there is someone with JS + Mobile experience out there that can provide some pointers, it will awesome.

**Comment by lazarusA** [comment]
Maybe relevant to the problem:

- https://discourse.threejs.org/t/touch-in-three-js/23382
- https://github.com/mese79/TouchControls
- https://stackoverflow.com/questions/59953438/three-js-touch-events-not-working-properly-on-mobile
- https://discourse.threejs.org/t/how-to-control-a-first-person-camera-on-touch-devices/19080

**Comment by lazarusA** [comment]
@asinghvi17 did you mention somewhere that now wglmakie works on phones? If that's the case, how did you tested it?

**Reply by asinghvi17** [reply]
I just loaded one of your examples (the image-on-Earth MPI ones) and it worked ;)

**Comment by BambOoxX** [comment]
I just tried to do similar things and now I'm facing the following problem.
- `text` plots in `:pixel` space are visible on both computer and mobile
- `text` plots in `:data` space are only visible on the computer. 

Can someone confirm this ? I'm working with the latest Makie (v0.22.10).
